templates/bindings/camel-k/avro-deserialize-action-binding.yaml

```diff
@@ -1,4 +1,4 @@
-# dont_overwrite
+# kamel bind --name avro-deserialize-action-binding timer-source?message='{"first":"Ada","last":"Lovelace"}' --step json-deserialize-action --step avro-serialize-action -p step-1.schema='{"type": "record", "namespace": "com.example", "name": "FullName", "fields": [{"name": "first", "type": "string"},{"name": "last", "type": "string"}]}' --step avro-deserialize-action -p step-2.schema='{"type": "record", "namespace": "com.example", "name": "FullName", "fields": [{"name": "first", "type": "string"},{"name": "last", "type": "string"}]}' --step json-serialize-action log:info
```

Comment:
Are these comments meant to be here? I am not sure about that "dont_overwrite" comment before.

Reply:
Yes, the comment is a marker to allow the generator.go to *not* generate the kamelet example file and the kamel bind example for the kamelet, and will source these examples into the .adoc files.